_events/2016-04-02.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Hip Hop Hacks"
+description: "On April 2nd and 3rd, we will be partnering with The Young Hackers to put on our first hip hop hackathon! We’ll be exploring and creating projects at the intersection of hip hop, technology and education."
+date: 2016-04-02 12:00pm
+days:
+    - day: 2016-04-02
+      start: 11:30am
+      end: 8:00pm
+    - day: 2016-04-03
+      start: 10:30am
+      end: 6:00pm
+address: "45 W 18th St, New York, NY 10011"
+rsvp: "https://www.eventbrite.com/o/monthly-music-hackathon-nyc-2470452960"
+image: #"place-this-image-file-in-assets-slash-events.jpg"
```

Comment:
Just out of curiosity, I checked. This does mean page.image returns false (which is of course good in this case). 

In general, I'd advocate for just removing anything commented out, so in this case, removing the whole line.